src/Domain/Students/StudentGateway.php

```diff
@@ -403,4 +403,19 @@ public function selectActiveStudentNames($gibbonSchoolYearID)
 
         return $this->db()->select($sql, $data);
     }
+
+    public function selectStudentEnrolmentHistory($gibbonPersonID)
+    {
+        $dataSelect = ['gibbonPersonID' => $gibbonPersonID];
```

Comment:
This is looking good, and great to see the SQL moved over into a Gateway class. Thanks for the quick work!

One very quick an minor change: when transplanting the SQL, be sure to simplify the $data and $sql variables, as they often have longer names like $dataSelect, etc because back in the day Ross was avoiding variable name conflicts with multiple queries in a script, but luckily Gateway classes solve this my using separate methods.